Most recently updated fork

pcouy opened the pull request:
I'd like to merge this but I am unable to test it (see the readme). If you are able to test it, please report back so that I know I can merge

kokonguyen191 committed:
Update AntiTranslate.user.js

kokonguyen191 committed:
Update AntiTranslate.user.js

kokonguyen191 committed:
Update AntiTranslate.user.js

kokonguyen191 committed:
Update README.md

kokonguyen191 committed:
Update README.md

kokonguyen191 committed:
Update AntiTranslate.user.js

kokonguyen191 committed:
Update AntiTranslate.user.js

kokonguyen191 committed:
Update AntiTranslate.user.js

kokonguyen191 committed:
Update AntiTranslate.user.js

kokonguyen191 committed:
Update README.md

kokonguyen191 committed:
Periodically check and refresh video title in case YT layout refreshed and updated it

adriaan1313 committed:
exclude playlists again

I assume this broke at some point, but now detecting if something is a playlist/mix based on the href rather than the classes

adriaan1313 committed:
Fix the search page and move namespace

- namespace move because the fork i forked this from (kokonguyen191/YoutubeAutotranslateCanceler) got deleted

adriaan1313 committed:
Stop "new" chrome update from fricking everything

* use "Trusted" HTML if needed
* bump version

adriaan1313 committed:
Fix removing contents of top comment

Fixes #1
If the video had no description, do not just go and assume the first text thingy is the description.

p-o-s commented:
seems to work on frontpage. 
doesn't show original title on ?watch= page.
changes excerpt of description to original language, but reverts to translation if you click on "...more"
does not change title attribute of `<a>`
does not change title on the "watch later" overview page, but does change it in the playlist panel to the right of video
didn't test anything else

adriaan1313 committed:
I hope this helps a bit

- Be more exact in playlist exclusions
- Exclude clips
- move a bit of the code that actually does the replacing of video titles and change it a bit in the hope that it may fix the problem

adriaan1313 committed:
Fix problem where links kept getting rechecked on the playlist page if switched over to from a different page

by checking if the link has any width/height at all

adriaan1313 committed:
bump version to 0.70

adriaan1313 committed:
version 0.70.0

- Hopefully fixes problem where things are mixed up with/appended to the wrong link.
- Reworks a bit of the code

adriaan1313 committed:
Update AntiTranslate.user.js

- stop leaking api key if someone has their devtools open and are screensharing/recording
- bump version to 0.70.1

adriaan1313 committed:
Fix full screen title

- and bump version to 0.70.2

adriaan1313 committed:
Update README.md

adriaan1313 committed:
Update howto to match how the google api console is now.

see #3

adriaan1313 committed:
Fix description on watch page

introduces even more duplication

adriaan1313 committed:
Fix descriptions on channel search pages

Fixes half of #4, sadly idk how to fix the other half yet

adriaan1313 committed:
Bump version to 0.70.3

adriaan1313 committed:
Merge pull request #6 from adriaan1313/fix-description-on-watch-page

Fixes #5 
- Also fixes half of #4 - descriptions on channel search and community pages.
- bumps version to 0.70.3

adriaan1313 committed:
Detranslate the videowall titles

- fix incorrect comment
- bump version to 0.70.4

adriaan1313 committed:
Update README.md

link to userscript was wrong

adriaan1313 committed:
fix description

- fix timestamps in description
- fix @tags in description
- fix #tags in description
- fix hash in url breaking current video id

adriaan1313 committed:
fix title tags

- fix # and @ tags in title

adriaan1313 committed:
bump version to 0.70.5

adriaan1313 committed:
Merge pull request #9 from adriaan1313/more-description-fixes

Fixes #8
- Fixes timestamps, and @ and # tags in description
- Fixes @ and # tags in title
- Fixes getting the proper id if there is a hash in the watch url
- Bumps version to 0.70.5

adriaan1313 committed:
Fix double timestamps in timestamp links

- Fix double timestamps in timestamp links in description
- Bumps version to 0.70.6

adriaan1313 committed:
fix title hash-tags properly
- Fix title hash-tags properly
- Bumps version to 0.70.7

adriaan1313 committed:
unbreak after recent yt update
- adds "yt-lockup-metadata-view-model__title" to span selector as youtube uses that sometimes now, fixing the watch and home pages
- bumps version to 0.70.8
(sorry for letting it be broken for a couple of days)

adriaan1313 committed:
Fix videowall
- Fixes videowall after todays yt update
- Bumps version to 0.70.9

adriaan1313 committed:
Fix issue where an error happens and breaks everyting

- Ignores the error so execution continues
- Bumps version to 0.70.10

adriaan1313 committed:
De-translate the links below the description

- De-translate the links below the description that used to be the info cards
- Bumps version to 0.70.11

Siegfried0 commented on README.md:
if you want to merge it back into pcouy:master, shouldn't you point the references back to pcouy's github?
in line 1 you even specify "notes exclusive to my fork".

I am not an developer and not that deeply into git, but this does not seem right to me.

pcouy replied:
I'm the one who created the pull request, and I'm okay with them making references to their own repo since I explicitly said I stopped maintaining my own version :)

adriaan1313 committed:
Things changed with the videos on the side

- Make sure they still get detranslated
- Bumps version to 0.70.12
